src/app/admin/admin.module.ts

```diff
@@ -4,7 +4,8 @@ import { CommonModule } from '@angular/common';
 
 import { AdminRoutingModule } from './admin-routing.module';
 import { UsersComponent } from './users/users.component';
-import { NgChartsModule } from 'ng2-charts';
+import { BaseChartDirective, provideCharts, withDefaultRegisterables } from 'ng2-charts';
+import { Legend, Colors } from 'chart.js';
```

Comment:
can be removed I guess...

Reply:
@whyvrafvr ?

Reply:
Sure let's merge and I'll remove these imports

Reply:
ok docky